docs/tm/udq_actionx/actionx.tex

```diff
@@ -342,24 +342,55 @@ \subsection{Updating simulator data structures}
 bugs in \actionx{} functionality have been in the interaction between the
 simulator and opm-common, in particular when an action has evaluated to true.
 
+An assumption permeating the simulator code is that changes to the well and
+group configuration only take place at report steps, and in between those the
+simulator ``owns'' the well and group data. Unfortunately this is no longer the
+case when \actionx{} is active, and depending on the keywords in the \actionx{}
+block we need to update the simulator data structures after \actionx{} has been
+evaluated to true. This complex, and most of the bugs in the \actionx{}
+implementation have been in this interface. There are probably more bugs waiting
+here.
```

Comment:
So does the simulator do this? Not really clear.

Reply:
Yes - but I anticipate there will be need for more such updates in the future. Have rewritten slightly.